osu.Game.Rulesets.Rush/Objects/Drawables/DrawableSawblade.cs

```diff
@@ -68,7 +68,7 @@ protected override void CheckForResult(bool userTriggered, double timeOffset)
             if (userTriggered || timeOffset < 0 || AllJudged)
                 return;
 
-            if (HitObject.HitWindows.CanBeHit(timeOffset))
+            if (HitObject.HitWindows.ResultFor(timeOffset) != HitResult.None)
                 return;
```

Comment:
This change got me really confused, what was the `CanBeHit` initially for, what does it change when using `ResultFor(...) != HitResult.None`?